parachain/pallets/omni-account/src/lib.rs

```diff
@@ -173,11 +174,15 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			member_account_hash: H256,
 			call: Box<<T as Config>::RuntimeCall>,
+			nonce: <T as frame_system::Config>::Nonce,
```

Comment:
It can be confusing if one only reads the parachain code, it's unclear why a `nonce` is required here (and what value it should be), AFAICT it's only relevant if we want to verify/add member account, right?

Shall we make it optional and add a comment when it's required? Like `maybe_expected_nonce: Option<Nonce>`

Reply:
This is actually the equivalent of the `nonce` check we currently do in the trusted call executor but now that I think about it we probably don't need this at all, what do you think?